Makefile

```diff
@@ -1147,72 +1159,62 @@ keystone_kuttl_run: ## runs kuttl tests for the keystone operator, assumes that
 keystone_kuttl: export NAMESPACE = ${KEYSTONE_KUTTL_NAMESPACE}
 # Set the value of $KEYSTONE_KUTTL_NAMESPACE if you want to run the keystone
 # kuttl tests in a namespace different than the default (keystone-kuttl-tests)
-keystone_kuttl: input openstack_crds deploy_cleanup mariadb mariadb_deploy mariadb_deploy_validate keystone_deploy_prep keystone ## runs kuttl tests for the keystone operator. Installs openstack crds and keystone operators and cleans up previous deployments before running the tests and, add cleanup after running the tests.
+keystone_kuttl: kuttl_db_prep keystone keystone_deploy_prep ## runs kuttl tests for the keystone operator. Installs keystone operator and cleans up previous deployments before running the tests, add cleanup after running the tests.
+	make wait
```

Comment:
To use `make wait`, `$(eval $(call vars,$@,keystone))` is needed here.  This is also true for all the other new additions of `make wait` that have been added in this PR (except, of course, `s/keystone/<operator_name>`).  Without it this error occurs:

```
Please set OPERATOR_NAME
+ exit 1
make[1]: *** [Makefile:364: wait] Error 1
```

Reply:
Oh, Thanks for pointing this. Fixed it in the latest version.